scvi/model/_autozi.py

```diff
@@ -177,7 +188,7 @@ def get_marginal_ll(
         )
 
         log_lkl = 0
-        to_sum = torch.zeros((n_mc_samples,))
+        to_sum = torch.zeros((n_mc_samples,)).to(self.device)
```

Comment:
@galenxing is there any reason this tensor was kept on cpu when you implemented it?